src/main/groovy/org/mockito/release/gradle/ReleaseNotesFetcherTask.java

```diff
@@ -174,6 +174,24 @@ public String getSkipCommitMessagePostfix() {
      * See {@link #getSkipCommitMessagePostfix()}
      */
     public void setSkipCommitMessagePostfix(String skipCommitMessagePostfix) {
+        /*
+
+        TODO mk - can you convert this to a list of ignored commit substrings?
+
+        I think the original design I put together has flaws.
+        commitMessagePostfix is dynamic and changes with every build because it contains Travis build number.
+        Therefore we cannot really depend on it to exclude commits reliably.
+
+        I suggest that we introduce a new setting to configure how to exclude commits from release generation.
+        This will make it explicit and easy to tweak by the user.
+        Suggested plan
+
+        1. New setting: "releasing.releaseNotes.ignoreCommitsContaining", Collection<String>
+        2. By default, we would configure it to: "ignoreCommitsContaining" = ['[ci skip]']
+        3. Users can tweak it. For example, in Mockito project, in "release.gradle" we would do:
+            releasing.releaseNotes.ignoreCommitsContaining = ['[ci skip]', '[ci skip-release]']
```

Comment:
Yes. It will be configurable. However, most users will not change the default values. We will preconfigure it to '[ci skip]'.

We don't want such settings to be hardcoded in the tools. There are potential use cases with different CI systems, different requirements where users might want to skip different commits. Also, there might be users who disagree that we should exclude '[ci skip]' because they want to include all commits in the notes.

Your work on this feature is an example of our direction - we build a flexible library :) When new use case / requirement gets in, we try to make the tool more flexible to support it.